### PR TITLE
Update index_en.rst

### PR DESCRIPTION
**Pull request by legend06hvl**
New file
fix https://github.com/PaddlePaddle/Paddle/issues/8915

### CODE REVIEW

File: doc/v2/dev/index_en.rst [hunk 1]
```diff
@@ -1,9 +1,27 @@
 Development
 ------------
 
+
+PaddlePaddle adheres to the following three sections of code and document specifications.
+
+
+PaddlePaddle uses git for version management and docker is used for building and testing environment. The code includes Cuda, C++, Python, Shell and other programming languages，which comply with Google C++ Style, Pep-8, and the code base includes style checking by an automatic inspection tool. Code comments need to follow the Doxygen specification, and if code that does not meet the style requirements will fail to compile with. we provide the following guidelines for the use of git, build tests and code development.
```
Comment (abhinavarora) [inline doc/v2/dev/index_en.rst]: version management -> version **control**

Comment (abhinavarora) [reply doc/v2/dev/index_en.rst]: docker -> **Docker**

Comment (abhinavarora) [reply doc/v2/dev/index_en.rst]: Doxygen specification, and if code that does not meet the style requirements will fail to compile with.  -> Doxygen specification. The code that does not meet the style requirements will fail to compile.

Comment (abhinavarora) [reply doc/v2/dev/index_en.rst]: we provide the -> **We** provide the

Comment (abhinavarora) [reply doc/v2/dev/index_en.rst]: git -> **Git**

File: doc/v2/dev/index_en.rst [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+..  toctree::
+  :maxdepth: 1
+
+  contribute_to_paddle_cn.md
+
+
+PaddlePaddle is suitable for domestic and foreign users, including Chinese and English documents. We recommend the English version of design documents and problem descriptions. The design documents focus on problem descriptions, backgrounds, and followed by solutions. As documents are generated by Sphinx, code comments should comply with the Sphinx document standard. We recommend to use the paddlepaddle.org tool to compile and generate and preview documents locally. Please refer to:
```
Comment (abhinavarora) [inline doc/v2/dev/index_en.rst]: I do not suggest to write the first sentence. The sentence makes PaddlePaddle sound like a localized project. We should instead write something like this:

_PaddlePaddle is well documented in English and Chinese. We recommend using the English version of the documents and problem description._

Comment (abhinavarora) [reply doc/v2/dev/index_en.rst]: The design documents focus on problem descriptions, backgrounds, and followed by solutions. -> The design documents focus on problem descriptions, backgrounds and are followed by solutions.

Comment (abhinavarora) [reply doc/v2/dev/index_en.rst]: Sphinx document standard -> Sphinx **documentation** standard

File: doc/v2/dev/index_en.rst [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+
+..  toctree::
+  :maxdepth: 1
+
+  write_docs_cn.rst
```
Comment (abhinavarora) [inline doc/v2/dev/index_en.rst]: I think this should be _write_docs_en.rst_

File: doc/v2/dev/index_en.rst [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
 
+
+PaddlePaddle adheres to the following three sections of code and document specifications.
+
+
+PaddlePaddle uses git for version management and docker is used for building and testing environment. The code includes Cuda, C++, Python, Shell and other programming languages，which comply with Google C++ Style, Pep-8, and the code base includes style checking by an automatic inspection tool. Code comments need to follow the Doxygen specification, and if code that does not meet the style requirements will fail to compile with. we provide the following guidelines for the use of git, build tests and code development.
+..  toctree::
+  :maxdepth: 1
+
+  contribute_to_paddle_cn.md
```
Comment (abhinavarora) [inline doc/v2/dev/index_en.rst]: This should be contribute_to_paddle_en.md

File: doc/v2/dev/index_en.rst [hunk 5]
```diff
@@ -1,9 +1,27 @@
...
+
+PaddlePaddle V2 defines new operations by adding new Layer. You can implement various complex layers by combining basic APIs to satisfy most applications. If you want to customize layer, please refer to the following, and welcome to propose patch.
+
 ..  toctree::
   :maxdepth: 1
 
-  contribute_to_paddle_en.md
-  write_docs_en.rst
-  new_layer_en.rst
+  new_layer_cn.rst
```
Comment (abhinavarora) [inline doc/v2/dev/index_en.rst]: I think this should be _new_layer_en.rst_

File: doc/v2/dev/index_en.rst [hunk 6]
```diff
@@ -1,9 +1,27 @@
...
+
+
+PaddlePaddle is suitable for domestic and foreign users, including Chinese and English documents. We recommend the English version of design documents and problem descriptions. The design documents focus on problem descriptions, backgrounds, and followed by solutions. As documents are generated by Sphinx, code comments should comply with the Sphinx document standard. We recommend to use the paddlepaddle.org tool to compile and generate and preview documents locally. Please refer to:
+
+..  toctree::
+  :maxdepth: 1
+
+  write_docs_cn.rst
+
+PaddlePaddle V2 defines new operations by adding new Layer. You can implement various complex layers by combining basic APIs to satisfy most applications. If you want to customize layer, please refer to the following, and welcome to propose patch.
```
Comment (abhinavarora) [inline doc/v2/dev/index_en.rst]: new Layer -> new **layers**.

**Comment by shanyi15** [comment]
Thank you so much for the review! @abhinavarora

**Pull request merged**